crates/cli/commands/src/test_vectors/compact.rs

```diff
@@ -94,7 +95,7 @@ compact_types!(
         Log,
         // BranchNodeCompact, // todo requires arbitrary
         TrieMask,
-        // TxDeposit, TODO(joshie): optimism
+        TxDeposit,
```

Comment:
```suggestion
```

I would rather remove this TODO, we should be special casing this in the op-reth test-vectors command, we also shouldn't enable the `op` reth-codec feature by default
https://github.com/paradigmxyz/reth/blob/81d76aa565df6a20dbc72d32f7ebf7c898f49332/crates/optimism/cli/src/commands/test_vectors.rs#L57-L61